src/WidgetUtils.js

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
```

Comment:
@dbkr - Vector creations 2017 was already on the next line, so I've just removed this one.